docs/src/components/CodePreview/CodePreview.component.tsx

```diff
@@ -166,29 +176,43 @@ export const CodePreview = ({
     // This is code taken from MDX's own documentation on rendering a code block
     // https://mdxjs.com/guides/syntax-highlighting/#build-a-codeblock-component
     return (
-        <Highlight
-            {...defaultProps}
-            code={children}
-            language={language}
-            theme={github}
-        >
-            {({ className, style, tokens, getLineProps, getTokenProps }) => (
-                <pre
-                    className={className}
-                    style={{ ...style, padding: '20px' }}
-                >
-                    {tokens.map((line, i) => (
-                        <div key={i} {...getLineProps({ line, key: i })}>
-                            {line.map((token, key) => (
-                                <span
-                                    key={key}
-                                    {...getTokenProps({ token, key })}
-                                />
-                            ))}
-                        </div>
-                    ))}
-                </pre>
+        <StyledContainerElement>
+            {!hideTitle && (
+                <Anchor.Typography tag="h6" pb="0" m="0" weight={600}>
+                    {title}
+                </Anchor.Typography>
             )}
-        </Highlight>
+
+            <Highlight
+                {...defaultProps}
+                code={children}
+                language={language}
+                theme={github}
+            >
+                {({
+                    className,
+                    style,
+                    tokens,
+                    getLineProps,
+                    getTokenProps,
+                }) => (
+                    <pre
+                        className={className}
+                        style={{ ...style, padding: '20px' }}
+                    >
+                        {tokens.map((line, i) => (
+                            <div key={i} {...getLineProps({ line, key: i })}>
```

Comment:
`key` shouldn't be the index value of a map function -sharkparty

Reply:
Spoke w/Kam on this. There isn't a great way to fix this particular issue and as it's static content it shouldn't matter much. Point taken for future usage.